chucks-pick3-web/angular-app/src/app/models/customer-info.model.ts

```diff
@@ -0,0 +1,35 @@
+import {CustomerInfo} from './customer-info';
+
+export class CustomerInfoModel implements CustomerInfo {
+  public firstName: string;
+  public lastName: string;
+  public email: string;
+  public phone: string;
+  public city: string;
+  public state: string;
+  public id: string;
```

Comment:
Please use Integer or int instead of string.

Reply:
Changed id value to 'number'.